WebUI/generate.sh

```diff
@@ -41,9 +42,9 @@ BUILD_DIR=$(pwd)/yaydoctemp
 
 cd $ROOT_DIR
 
-cp -a ../../../scripts/ $BUILD_DIR/
-cp -a ../../../templates/ $BUILD_DIR/
-cp -a ../../../requirements.txt $BUILD_DIR/
+cp -a ../../../../scripts/ $BUILD_DIR/
+cp -a ../../../../templates/ $BUILD_DIR/
+cp -a ../../../../requirements.txt $BUILD_DIR/
```

Comment:
You don't need to. It's not all your fault. I was supposed to test the PR. Saw that the exit code returned by the script was 0 and thought it would be correct. 
It was later when I checked the logs properly, i found the bug.